pkgs/by-name/ho/honeyfetch/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  libdrm,
+  fetchFromGitLab,
+  rustPlatform,
+}:
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "honeyfetch";
+  version = "1.5.0";
+
+  src = fetchFromGitLab {
+    owner = "ahoneybun";
+    repo = "honeyfetch";
+    tag = "v${finalAttrs.version}";
+    sha256 = "sha256-lFp7L3tcqZ1jAL7V7tfUJDPKO2WCvMekUx+p12fNlcM=";
+  };
+
+  cargoHash = "sha256-u+OF4ali7GoHktY8jihgqUQ+4kFuKQNbiaRUdOJrQfA=";
+
+  buildInputs = [libdrm];
+
+  meta = {
+    description = "A classy neofetch but in Rust";
```

Comment:
Starting with `A ` seems not be fit with this guide

https://github.com/NixOS/nixpkgs/blob/66023e4de2495a69792a2b72bd131358b824d2e3/pkgs/README.md?plain=1#L472C4-L472C58